examples/run/README.md

```diff
@@ -18,7 +18,9 @@ Options:
   -c, --context-size <value>
       Context size (default: 2048)
   -n, --ngl <value>
-      Number of GPU layers (default: 0)
+      Number of GPU layers (default: 999)
```

Comment:
I don't think this is accurate, the default is still 0.

Reply:
If I print this usage help on a mac it's 999, if I print this usage help on a non-mac, it's 0, so both are correct, it prints something different per platform because there's an ifdef if you compile with metal or not. Which I think is probably best, everyone on a Apple Silicon machine should be running accelerated except for exceptional circumstances.

Reply:
Anyway, I'll set it to 0, it doesn't really matter

Reply:
Done